python/ray/tests/test_ray_init.py

```diff
@@ -63,7 +63,7 @@ def test_redis_port(self, shutdown_only):
         def f():
             return 1
 
-        info = ray.init(redis_port=1234, redis_password="testpassword")
+        info = ray.init(redis_port=1234, _redis_password="testpassword")
```

Comment:
Ah was this a new decision? It actually seems a bit strange to me to have `redis_port` but then a hidden `_redis_password`.

Reply:
We don't support redis_port any more; removing this test.